Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+
+The two fundamental subjects to know in order to develop successful Data Science projects are **descriptive statistics** and **theory of probability**.
+
+We'll explore an additional one, Bayesian Statistics, which is pretty powerful but not mandatory for a first pass of the statistics topics.
+
+Play with [this interactive website](https://seeing-theory.brown.edu/) to taste the flavor of the topics that this guide will suggest to you.
+
+It's awesome.
+
+### Descriptive Statistics
+
+[Descriptive statistics](https://www.investopedia.com/terms/d/descriptive_statistics.asp) aim to analyze and summarise the data collected in an experiment. 
+
+The set of methods and techniques used in descriptive statistics makes it possible to express the information contained in a set of data by means of graphs and particular numerical indicators.
+
+Moreover, with the techniques of investigation of descriptive statistics, it is possible to verify the adaptation of the experimental data to an assigned theoretical model, as well as to carry out comparative analyses between data sets.
+
+Descriptive statistics are widely used in economics, demography, medicine, and in all-natural sciences.
+
+First of all watch these videos, they are truly inspirational:
+
+- [Why You Should Love Statistics](https://www.ted.com/talks/alan_smith_why_we_re_so_bad_at_statistics)
+- [The Best Stats You've Ever Seen](https://www.ted.com/talks/hans_rosling_shows_the_best_stats_you_ve_ever_seen)
+
+---
+
+To understand the concepts of descriptive statistics, the book Virgilio recommends is the following:
+
+[**Statistics in Plain English**](http://www.pindex.com/uploads/post_docs/statistics%20in%20plain%20english%20(3rd,%202010)(PINDEX-DOC-6952).pdf) - [Timothy C. Urdan](https://www.amazon.com/Timothy-C.-Urdan/e/B001JP8H9Q%3Fref=dbs_a_mng_rwt_scns_share)
+
+This book is fantastic because it focuses more on **the reason behind the concepts** concepts rather than on formulas or calculations, which are easily found elsewhere. 
+
+**The book is mandatory for a full understanding of descriptive statistics** (it is certainly not the only book of statistics out there, but it is the best in terms of clarity of explanation and completeness of the basic concepts.).
+
+Once finished, you can choose whether to go deeper with a more technical manual, like the following _free book_:
+
+[**Elements of Statistics**](https://www.open.edu/openlearncreate/mod/oucontent/view.php?id=18263&printable=1)
+
+This book deals in more depth with the various topics, but to start doing small experiments in Data Science is not necessary to address it immediately. 
+
+**Keep it as a reference!**
+
+#### Check your knowledge
+
+Test your knowledge with some of [these exercises](https://www0.gsb.columbia.edu/faculty/pglasserman/B6014/exercises.pdf).
+
+Anyway, you can find a ton of them online, and remember, that the more you practice, the better you will get!
+
+
```

Comment:
Probably nice to mention, just for the sake of completeness, is the book ["How to lie with Statistics"](https://en.wikipedia.org/wiki/How_to_Lie_with_Statistics) that shows how important it is that the general public be educated when it comes to statistics. Moreover if you plan to fiddle around with data and plots and whatnot!

Reply:
Totally true! I 've forgotten it. thanks